app/src/main/java/org/hackillinois/android/viewmodel/ScannerViewModel.kt

```diff
@@ -20,17 +20,29 @@ import retrofit2.Response
 class ScannerViewModel : ViewModel() {
     private val eventRepository = EventRepository.instance
     var eventsListLiveData: LiveData<List<Event>> = MutableLiveData<List<Event>>()
-    var lastScanStatus: MutableLiveData<ScanStatus> = MutableLiveData<ScanStatus>()
+    var lastScanStatus: MutableLiveData<ScanStatus> = MutableLiveData()
     var shouldDisplayOverrideSwitch = MutableLiveData<Boolean>()
 
-    val CHECK_IN_TEXT = "Check In"
+    private val CHECK_IN_TEXT = "Check In"
 
     fun init() {
         eventsListLiveData = eventRepository.fetchAllEvents()
         // Hidden by default
         shouldDisplayOverrideSwitch.postValue(false)
     }
 
+    fun checkUserIntoEvent(eventName: String, userId: String, staffOverride: Boolean) {
+        if (eventName == CHECK_IN_TEXT) {
+            val hasCheckedIn = true
+            val hasPickedUpSwag = true
```

Comment:
Doesn't `hasCheckedIn` imply `hasPickedUpSwag`? Do we need two separate variables here?

Reply:
I think this was resolved in the other PR comments, so I'll close this.